sale_order_import/models/sale_import_default.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleImportDefault(models.Model):
+    _name = "sale.import.default"
+
+    company_id = fields.Many2one(
+        "res.company",
+        required=True,
+        string="Company",
+    )
+    picking_policy = fields.Selection(
+        [
+            ("direct", "Deliver each product when available"),
+            ("one", "Deliver all products at once"),
```

Comment:
```suggestion
            ("direct", "As soon as possible"),
            ("one", "When all products are ready"),
```
Following https://github.com/odoo/odoo/blob/51ff14a56b909bdf22c9f1c4540347655578e1f4/addons/point_of_sale/models/pos_config.py#L164-L165.

I wonder if there is a nice way to take the selection optisons from an existing field.